perun-core/src/main/java/cz/metacentrum/perun/core/entry/MembersManagerEntry.java

```diff
@@ -1454,14 +1454,21 @@ public RichMember sponsorMember(PerunSession session, Member sponsored, User spo
 
 	@Override
 	public List<MemberWithSponsors> getSponsoredMembersAndTheirSponsors(PerunSession sess, Vo vo, List<String> attrNames) throws VoNotExistsException, PrivilegeException, AttributeNotExistsException {
```

Comment:
Please keep the old politic `getSponsoredMembersAndTheirSponsors_Vo_policy` for this method. It is because the GUI needs it to resolve authorization.

Reply:
Fixed